app/src/js/actions/index.js

```diff
@@ -542,14 +545,17 @@ export const getCount = (options) => ({
   }
 });
 
-export const listPdrs = (options) => ({
-  [CALL_API]: {
-    type: types.PDRS,
-    method: 'GET',
-    url: url.resolve(root, 'pdrs'),
-    qs: Object.assign({ limit: pageLimit }, options)
-  }
-});
+export const listPdrs = (options) => {
+  return (dispatch, getState) => {
+    const timeFilters = fetchCurrentTimeFilters(getState().datepicker);
+    dispatch({[CALL_API]: {
```

Comment:
I could be wrong, but I think you want to `return dispatch`, otherwise you wouldn't be able to do `.then` on `listPdrs` or any of these action creators if you wanted to later

Reply:
You're right.  a52de30f21f028c18a790305912b8af3177cd036